### ISSUE TITLE
Right-click export does not work

### ISSUE THREAD

**Issue by YuuuuXie** [shd101wyy/vscode-markdown-preview-enhanced#194]
I tried to right click to export my .md to html or pdf. The menu "open in browser, etc...." appears, but  when I click any option, there's no response. I'm using Mac, it there any shortcut?

(Also, I'm trying presentation, so it seems that the export_on_save also does not work)

**Comment by shd101wyy** [comment]
Yes I am working on a new release now, which will fix the problems that you were facing.

https://github.com/shd101wyy/vscode-markdown-preview-enhanced/pull/193

Ideally this PR could be done tomorrow.
Thanks

**Comment by YuuuuXie** [comment]
It seems that the right-click problem has been solved. Thx! 
But I got another issue for using 'open in browser' or 'save to html'. They work fine for non-presentation mode. But for presentation, when I open in browser, there's no text, only an empty background. (Though pdf prince is fine.)

**Comment by w2816771** [comment]
> It seems that the right-click problem has been solved. Thx!
> But I got another issue for using 'open in browser' or 'save to html'. They work fine for non-presentation mode. But for presentation, when I open in browser, there's no text, only an empty background. (Though pdf prince is fine.)
> 
> 'html csd hosted' works, but 'html offline' does not

have the same issue here
